 Fix wifi Supplicant Scanner
 The results given to Wifi QNS when scanning after deep sleep is empty thus Wifi Autoconnection fails. Hack this thing.

diff --git a/service/java/com/android/server/wifi/scanner/SupplicantWifiScannerImpl.java b/service/java/com/android/server/wifi/scanner/SupplicantWifiScannerImpl.java
index ac5db5a..e8faaba 100644
--- a/service/java/com/android/server/wifi/scanner/SupplicantWifiScannerImpl.java
+++ b/service/java/com/android/server/wifi/scanner/SupplicantWifiScannerImpl.java
@@ -556,7 +556,7 @@ public class SupplicantWifiScannerImpl extends WifiScannerImpl implements Handle
             for (int i = 0; i < nativeResults.size(); ++i) {
                 ScanResult result = nativeResults.get(i).getScanResult();
                 long timestamp_ms = result.timestamp / 1000; // convert us -> ms
-                if (timestamp_ms > mLastScanSettings.startTime) {
+                if (timestamp_ms > mLastScanSettings.startTime || true) {
                     if (mLastScanSettings.backgroundScanActive) {
                         backgroundScanResults.add(result);
                     }
